src/ncorr_commandLine.cpp

```diff
@@ -797,13 +687,32 @@ int main(int argc, char *argv[]) {
 	}
 
 	else {
-		if(isEulerian)
-			saveImages(videoExport, strain_input_eulerian, getImagesArray, videoPath, strainType, DIC_output_eulerian, strain_output_eulerian, strainMax);
+		if(eulerian_video)

```

Comment:
Prefer to have {} on this `if` statement

Reply:
Yeah, I guess I find it 'cute', but consistency is probably more important.